pkg/system/system.go

```diff
@@ -624,6 +636,37 @@ func RunStatus(cmd *cobra.Command, args []string) {
 
 }
 
+// RunDump generates a tar file of a db dump
+func RunSetDebugLevel(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	level := 0
+	if args[0] == "warn" {
+		level = -1
+	} else {
+		var err error
+		level, err = strconv.Atoi(args[0])
+		if err != nil || level < 0 || level > 5 {
+			log.Fatalf(`invalid debug-level argument. must be 'warn' or 0 to 5. %s`, cmd.UsageString())
+		}
+	}
+	nbClient := GetNBClient()
+	err := nbClient.PublishToCluster(nb.PublishToClusterParams{
+		Target:     "",
+		MethodAPI:  "debug_api",
+		MethodName: "set_debug_level",
+		RequestParams: nb.SetDebugLevelParams{
+			Module: "core",
```

Comment:
don't we want to give an advance option to change the module? 
and root to core as a default? 
for example:
```
Module: "core.endpoint.s3"
```
will only set the endpoint.s3 to the desired log level.

Reply:
I think that for now, I will leave it global (`core` module). we can add more fine-grain configuration later if we see a need